Modules/storage.rb

```diff
@@ -0,0 +1,17 @@
+class Storage
+  @file_extention = 'json'
+  @base_url = './db/'
+
+  def self.save_date(class_name, object)
+    File.write("#{@base_url}#{class_name}.#{@file_extention}", JSON.generate(object))
+  end
+
+  def self.load_data(class_name)
+    file_path = "#{@base_url}#{class_name}.#{@file_extention}"
+
+    return [] unless File.exist?(file_path)
+
+    data_from_file = File.read(file_path)
+    JSON.parse(data_from_file, create_additions: true)
+  end
+end
```

Comment:
- [ ] On exit from the app the data should be saved (all the data: books, people, and rentals) into JSON files. One per type of data should be stored and named accordingly (e.g. rentals.json). But your app is broken, when I try to create people, books or rentals I get an error in the terminal. Please kindly fix that and make sure your data are preserved correctly 🙃 🙃 

<img width="1082" alt="Screen Shot 2023-01-18 at 18 32 08" src="https://user-images.githubusercontent.com/74501523/213222941-0e0dd820-a4fd-46b8-84ea-193d7a5b6b6d.png">

Reply:
- [ ] Please fix this as requested by the previous reviewer. I noticed you had not made changes. When I successfully create a person, book or rental, an error is thrown in the terminal because you are trying to save the data to a directory that does not exist(`./db/Person.json`). Please make sure you save the data to an existing directory successfully.

Reply:
@cynthiainga @Lucas-Masaba all i had to do is add db folder.